Dapper/CustomPropertyTypeMap.cs

```diff
@@ -29,13 +29,13 @@ public CustomPropertyTypeMap(Type type, Func<Type, string, PropertyInfo> propert
         /// <param name="types">DataReader column types</param>
         /// <returns>Default constructor</returns>
         public ConstructorInfo FindConstructor(string[] names, Type[] types) =>
-            _type.GetConstructor(Array.Empty<Type>());
+            _type.GetConstructor(Array.Empty<Type>())!;
 
         /// <summary>
         /// Always returns null
         /// </summary>
         /// <returns></returns>
-        public ConstructorInfo FindExplicitConstructor() => null;
+        public ConstructorInfo FindExplicitConstructor() => null!;

```

Comment:
This...seems like a lie?

Reply:
I... hmmm, I can't fault you. I'm curious why this is. Not why this is anything specific, just: *why this is*